### PR TITLE
ci(github): update `figma` tokens workflow

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

$subject with the new PAT.

### Related Issues
- None

### Related PRs
- None

### Checklist
- [ ] UX/UI review done on the final implementation.
- [ ] Story provided. (Add screenshots)
- [ ] Manual test round performed and verified.
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Documentation provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .github/workflows/figma-tokens-update-trigger.yaml [hunk 1]
```diff
@@ -25,5 +25,16 @@ jobs:
...
+          pr_body: |                                        # Full markdown support, requires pr_title to be set
+            :crown: *An automated PR*
+
+            Design Tokens were updated in Figma!
+
+            This PR was created to update the corresponding style dictionary tokens.
+
+            ## Process
+
+            - Figma tokens will be updated in primitives. Click [here →](https://github.com/wso2/oxygen-ui/tree/main/packages/primitives/figma) to inspect the exiting tokens.
```
Comment (savindi7) [inline .github/workflows/figma-tokens-update-trigger.yaml:37]:
```suggestion
            - Figma tokens will be updated in primitives. Click [here →](https://github.com/wso2/oxygen-ui/tree/main/packages/primitives/figma) to inspect the existing tokens.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -251,6 +251,41 @@ The content of the commit message body should contain:
...
+2. Use the Figma Tokens plugin to push the tokens directly from Figma to the Oxygen UI repository.
+3. Commit and push the token changes to GitHub, using the following commit message:
+
+  ```bash
+  chore(figma): sync figma tokens with github
+  ```
+
+  Above will commit directly to the [`figma-tokens`](https://github.com/wso2/oxygen-ui/tree/figma-tokens) branch.
+
+4. A GitHub workflows will automatically be triggered to process the commited design tokens and convert them to a [style dictionary](https://github.com/amzn/style-dictionary) compatible design-token variation.
```
Comment (savindi7) [inline CONTRIBUTING.md:270]:
```suggestion
4. A GitHub workflows will automatically be triggered to process the committed design tokens and convert them to a [style dictionary](https://github.com/amzn/style-dictionary) compatible design-token variation.
```

**Review by savindi7 — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (brionmario) [reply CONTRIBUTING.md]: Fixed with https://github.com/wso2/oxygen-ui/pull/74/commits/560999f4994af32568a3f2098d4b8efd1885f63c. Thanks 🙌.

File: .github/workflows/figma-tokens-update-trigger.yaml (hunk 1 above)
Comment (brionmario) [reply .github/workflows/figma-tokens-update-trigger.yaml]: Fixed with https://github.com/wso2/oxygen-ui/pull/74/commits/560999f4994af32568a3f2098d4b8efd1885f63c. Thanks 🙌.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -251,6 +251,41 @@ The content of the commit message body should contain:
...
+4. Once the workflow finishes, a PR will be created against the `main` branch with the following format.
+
+  ```bash
+  # PR Title
+  chore(primitives): update figma tokens
+
+  # PR Lables
+  `automated pr`, `package:primitives`
+  ```
+5. Review the PR and merge it to the `main` branch.
```
[the remark this answers is not in the document]
Comment (brionmario) [reply CONTRIBUTING.md]: Fixed with https://github.com/wso2/oxygen-ui/pull/74/commits/560999f4994af32568a3f2098d4b8efd1885f63c. Thanks 🙌.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -251,6 +251,41 @@ The content of the commit message body should contain:
...
+
+  ```bash
+  chore(figma): sync figma tokens with github
+  ```
+
+  Above will commit directly to the [`figma-tokens`](https://github.com/wso2/oxygen-ui/tree/figma-tokens) branch.
+
+4. A GitHub workflows will automatically be triggered to process the commited design tokens and convert them to a [style dictionary](https://github.com/amzn/style-dictionary) compatible design-token variation.
+
+4. Once the workflow finishes, a PR will be created against the `main` branch with the following format.
```
[the remark this answers is not in the document]
Comment (brionmario) [reply CONTRIBUTING.md]: Fixed with https://github.com/wso2/oxygen-ui/pull/74/commits/560999f4994af32568a3f2098d4b8efd1885f63c. Thanks 🙌.

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**